portfolio/src/main/java/com/google/sps/data/PresonIn.java

```diff
@@ -0,0 +1,30 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class PresonIn {
+    private String name;
+    private int age;
+    public String address;
+
+public void setName(String a)
```

Comment:
Since the function is so short, it makes sense to want to use a short variable name for this argument. Even if it's only one letter though, consider having it correspond in some way to the value it represents. Since it's the name in this function, consider having it be "n" (as in "name") instead of just "a".